package.json

```diff
@@ -43,6 +43,7 @@
     }
   ],
   "activationEvents": [
+    "*",
```

Comment:
if you add `*` are others still needed? why is this necessary?

Reply:
The others are not strictly necessary. I kept them so that when we figure out how to not use `"*"` here we can just remove it.

The reason is that if Joyride started later in the process, its output panel would show regardless of us showing it or not. As a result of creating it. I don't know why this happens. Some strange decision at MS, or if it is a bug.